packages/pitop/README.rst

```diff
@@ -126,57 +126,84 @@ Check out the `CLI`_ chapter of the documentation for ideas on how to get starte
 Installation
 ------------
 
+OS
```

Comment:
I don't see why we should have a full repeat of the main readme in the main subpackage readme here, unless it's getting installed from here or it's for the pypi page or something along those lines?

I think it would be best to just have a short description like the other subpackages. eg "This is the top level package that collects together the subpackages, plus a few extras like 'labs'."

Maybe you could reference the main readme for people who are looking for that. (If you do this maybe put a full github url rather than just relative path for scenarios like pypi page)

Reply:
yeah, I didn't remove it since it's what someone will see when going to the `pitop` package in pypi. I'll keep the images and update the description to explain what that package in particular does.